content/agent/basic_agent_usage/_index.md

```diff
@@ -1,4 +1,96 @@
 ---
-title: Basic Agent Usage
-external_redirect: /agent/basic_agent_usage/amazonlinux/
+title: Basic Agents Usage
+kind: documentation
+aliases:
+    - /guides/basic_agent_usage/
+    - /agent/faq/where-is-the-configuration-file-for-the-agent/
+    - /agent/faq/log-location
 ---
+
+{{< partial name="platforms/platforms.html" >}}
+
+## Configuration management tools
+
+Manage the Datadog Agent and [integrations](/integrations) using configuration management tools:
+
+### Chef
+* [Chef Github project](https://github.com/DataDog/chef-datadog)
+* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
+
+### Puppet
+* [Puppet Github project](https://github.com/DataDog/puppet-datadog-agent)
+* [Puppet installation](https://app.datadoghq.com/account/settings#integrations/puppet)
+
+### Ansible
+* [Ansible Github project](https://github.com/DataDog/ansible-datadog)
+* [Ansible installation](https://app.datadoghq.com/account/settings#agent/ansible)
+
+Chef, Puppet, and Ansible [integrations](/integrations) use our public APIs, so if you're interested in using another automation tool, the above could be leveraged as examples to get you started.
+
+There is also community support for Saltstack:
```

Comment:
we could list the SaltStack formula as "officially" supported at this point, I took over repo maintenance and released a version 1.0 only supporting A5. Version 2.0 will be released soon to add support for A6.
Nit: it's `SaltStack`, camelcase

Reply:
👌